### DISCUSSION TITLE
what diff rxinfer with https://github.com/Juice-jl/Dice.jl/

### DISCUSSION THREAD

**Discussion by zdx3578** [ReactiveBayes/RxInfer.jl#168] [Q&A]
what diff rxinfer with https://github.com/Juice-jl/Dice.jl/

**Comment by albertpod** [comment]
From dice GitHub:
`dice is a probabilistic programming language focused on fast, exact inference for discrete probabilistic programs`
RxInfer.jl focuses on reactive message passing for exact and approximate inference in hybrid models containing discrete and continuous RVs.
